test/PrintAsCxx/empty.swift

```diff
@@ -0,0 +1,54 @@
+// RUN: %empty-directory(%t)
```

Comment:
Does it make sense to put this in `Interop`? Like `test/Interop/Swift` or something?

Reply:
Hmm, not sure. I think something like `test/Interop/Swift` doesn't make the most sense as it implies Swift is interoperating with itself. I think sticking to `PrintAsCxx` might make more sense.